osmtm/utils.py

```diff
@@ -109,3 +111,19 @@ def convert_to_multipolygon(geoms):
     geom2d = shapely.wkt.loads(wkt2d)
 
     return geom2d
+
+
+def interpolate_text(text, properties):
```

Comment:
Hah - thank you for the catch there, @iandees - bit silly of me there - fixed in https://github.com/hotosm/osm-tasking-manager2/pull/767/commits/89b9ad4b686f7250c66a6f9f13946ccd10139557